Source/Particles/ElementaryProcess/QEDInternals/BreitWheelerEngineWrapper.cpp

```diff
@@ -192,7 +192,7 @@ void BreitWheelerEngine::init_builtin_pair_prod_table()
     pair_prod_params.chi_phot_how_many = 64;
     pair_prod_params.frac_how_many = 64;
 
-    const auto vals = amrex::Gpu::DeviceVector<amrex::Real>{
+    const auto vals = amrex::Gpu::DeviceVector<amrex::ParticleReal>{
```

Comment:
```suggestion
    const auto vals = amrex::Gpu::DeviceVector<amrex::ParticleReal>{
```

Keep this as `Real`?